lms/static/js/views/image_field.js

```diff
@@ -208,9 +209,9 @@
             onBeforeUnload: function () {
                 var status = this.getCurrentStatus();
                 if (status === 'uploading') {
-                    return gettext("Upload is in progress. To avoid errors, stay on this page until the process is complete.");
+                    return gettext('Upload is in progress. To avoid errors, stay on this page until the process is complete.');  // jshint ignore:line
```

Comment:
This is a browser specific event that shows the returned text in a dialog. It doesn't seem to be HTML and so maybe doesn't need escaping...

https://developer.mozilla.org/en-US/docs/Web/Events/beforeunload